backend/otel/otel.go

```diff
@@ -263,7 +263,6 @@ func (o *Handler) HandleTrace(w http.ResponseWriter, r *http.Request) {
 						logRow := clickhouse.NewLogRow(
 							fields.timestamp, uint32(fields.projectIDInt),
 							clickhouse.WithTraceID(traceID),
-							clickhouse.WithSpanID(spanID),
```

Comment:
Seems like we should not include these since they will not have a related span in the traces search